package pe.uni.poo_v_g7.bibliotecaapp.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import pe.uni.poo_v_g7.bibliotecaapp.entity.Cliente;

public interface ClienteRepository extends JpaRepository<Cliente, Integer> {
}
